tasks/install-plugins.yml

```diff
@@ -27,20 +27,39 @@
       # TODO: Remove this suppression when ansible-lint learns to parse block conditions
       # See: https://github.com/ansible/ansible-lint/issues/1894
       # noqa no-changed-when
-      ansible.builtin.command: >
+      ansible.builtin.shell: >
         java -jar
         {{ jenkins_pimt_path }}
         --plugin-download-directory {{ jenkins_plugins_dir }}
         --plugin-file {{ jenkins_config_dir }}/plugins.yaml
         --war {{ jenkins_war }}
         --verbose
         {{ jenkins_pimt_args }}
+        {{ (jenkins_pimt_logfile != "") | ternary("2> /tmp/pimt.log", "") }}
       environment:
         CACHE_DIR: "{{ jenkins_caches_dir }}"
+      # Weirdly, no_log wants to interpret `jenkins_pimt_logfile != ''` as a string, not
+      # a boolean. So we need to use the ternary filter again.
+      no_log: "{{ (jenkins_pimt_logfile != '') | ternary(true, talse) }}"
```

Comment:
talse?

Reply:
That's the problem with the verify tests examining the `true` case. :sweat_smile: Fixed in 9e8e912.